scripts/ddtest

```diff
@@ -31,8 +31,8 @@ if [[ "${CIRCLECI}" = "true" ]]; then
                    bash -c "$FULL_CMD"
 else
     docker-compose run \
-                   -e DD_TRACE_AGENT_URL \
```

Comment:
why are we able to remove this?

Reply:
Does the testrunner default to port 9126?

Reply:
That was a mistake. Thanks for catching it!